wrappers/python/simtk/openmm/app/internal/customgbforces.py

```diff
@@ -377,36 +393,59 @@ def _createEnergyTerms(force, solventDielectric, soluteDielectric, SA, cutoff, k
         E.sulfur : (0.96, 0.602256336067, -0.703469),
         None : (0.8, 0.5, 0.5) # default
 }
+
 _SCREEN_PARAMETERS[E.deuterium] = _SCREEN_PARAMETERS[E.hydrogen]
 def _screen_parameter(atom):
     if atom.element in _SCREEN_PARAMETERS:
         return _SCREEN_PARAMETERS[atom.element]
     return _SCREEN_PARAMETERS[None]
 
+
 class CustomAmberGBForce(CustomGBForce):
+    """Base class for all of the Amber custom GB forces.
+
+    Cannot be instantiated.
+
+    """
+    __metaclass__ = abc.ABCMeta
```

Comment:
You can't use metaclasses in OpenMM without adding the `six` module.  Python 3 adds metaclasses via a `metaclass=` argument to the class declaration.  Without running this code through `2to3`, Python 3 will choke on this.

If you have `six`, you can do `@add_metaclass(abc.ABCMeta)` in the class decorator.  But that's why I didn't consider doing anything like this before.  Is this level of complexity really necessary?  Or can we get rid of the abstract base class stuff and go back to how it was?